admin/google_search_console/class-gsc.php

```diff
@@ -81,7 +84,7 @@ public function register_gsc_notification() {
 					'</a>'
 				),
 				array(
-					'type'      => 'updated yoast-dismissible',
+					'type'      => 'warning',
```

Comment:
See my comment about the `error` constant. If you decide to do that, it should be done also for `warning`